packages/ui/react-ui/src/CrossmintNFTCollectionView.tsx

```diff
@@ -0,0 +1,30 @@
+import * as React from "react";
+
+import {
+    NFTCollectionViewProps,
+    assertValidNFTCollectionViewProps,
+    getNFTCollectionViewSrc,
+} from "@crossmint/client-sdk-base";
+
+import { LIB_VERSION } from "./version";
+
+export function CrossmintNFTCollectionView(props: NFTCollectionViewProps) {
+    assertValidNFTCollectionViewProps(props);
+
+    const src = getNFTCollectionViewSrc(props, LIB_VERSION);
+
+    return (
+        <iframe
+            src={src}
+            width="100%"
+            height="100%"
+            style={{
```

Comment:
its not `display: flex` so flex grow doesnt do anything i think

Reply:
Actually, the display flex would be on the parent